app/src/lib/git/status.ts

```diff
@@ -35,7 +35,33 @@ export interface IStatusResult {
   readonly workingDirectory: WorkingDirectoryStatus
 }
 
-function convertToAppStatus(status: FileEntry): AppFileStatus {
+async function entryHasConflictMarkers(
+  repositoryPath: string,
+  entry: IStatusEntry,
+  status: FileEntry
+): Promise<boolean | null> {
+  // only conflicted files can have conflict markers
+  if (status.kind !== 'conflicted') {
+    return null
```

Comment:
Any reason why we can't return `false` here?

Reply:
i was thinking that detecting the presence or lack of conflict markers doesn't mean anything if the file isn't `'conflicted'`, so `null` could represent that case. but i'm not sure that that information is actually useful so i'm good with `false` here. 064a919